### PR TITLE
Frontend added

### PR DESCRIPTION
**Pull request by EGheska**
React application was created.
Node modules doesn't included
src contains components and .js files contains pages
index.html is filled

### CODE REVIEW

**Review by jurijjzz — APPROVED** [review]

File: frontend/src/App.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+const App = () => {
+    const {UserStore} = useContext(Context);
+    console.log(UserStore);
+    return (
+        <div className="App">
+                <Navbar/>
+        </div>
+    )
+};
+export default App;
```
Comment (06Pac) [inline frontend/src/App.js]: Wrap app into observer

File: frontend/src/components/header/navbar.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+                    <div className="dropdown">
+                        <button className="dropbtn"></button>
+                        <div className="dropdown-content">
+                            <a href="#" id="link">Profile</a><a href="#" id="link">Logout</a>
+                            <a href="#" id="link">Help</a>
+                        </div>
+                    </div>
+                ) : (
+                    <div>
+
```
Comment (06Pac) [inline frontend/src/components/header/navbar.js]: Add second option, when user is not logged in

**Review by 06Pac — CHANGES_REQUESTED** [review]

File: frontend/package-lock.json
Comment (NeuroCorgi) [inline frontend/package-lock.json:1]: Remove this file, it is not needed

File: frontend/public/favicon.ico
Comment (NeuroCorgi) [inline frontend/public/favicon.ico:1]: Is it really needed?

File: frontend/src/components/body/body.jsx
Comment (NeuroCorgi) [inline frontend/src/components/body/body.jsx:1]: Maybe add more semantic tags?

File: frontend/src/components/body/body_style.css
Comment (NeuroCorgi) [inline frontend/src/components/body/body_style.css:1]: Think about switching to tailwind or (forgive me) ~~bootstrap~~

File: frontend/src/components/header/header.js
Comment (NeuroCorgi) [inline frontend/src/components/header/header.js:1]: Eliminate dead code

File: frontend/src/components/header/style.css
Comment (NeuroCorgi) [inline frontend/src/components/header/style.css:1]: Same as main css. Think about tailwind(~~bootstrap~~)

**Review by NeuroCorgi — CHANGES_REQUESTED** [review]

**Review by jurijjzz — APPROVED** [review]

**Review by jurijjzz — CHANGES_REQUESTED** [review]
fix it please

**Review by 06Pac — COMMENTED** [review]
Make logic names for component folders like section, bars, menu ......

File: frontend/src/pages/user.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import {Body} from "../components";
+
+
```
Comment (06Pac) [inline frontend/src/pages/user.js]: Change type to jsx

**Review by 06Pac — COMMENTED** [review]
Make logic names for component folders like section, bars, menu ......

File: frontend/package-lock.json
Comment (NeuroCorgi) [inline frontend/package-lock.json:1]: Remove this file

**Review by NeuroCorgi — CHANGES_REQUESTED** [review]
New files in 'forms' directory are empty

**Review by jurijjzz — APPROVED** [review]

**Review by NeuroCorgi — APPROVED** [review]

**Review by 06Pac — APPROVED** [review]

**Pull request merged**